src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -181,7 +186,7 @@ public void run() {
    * </code></pre>
    */
   private static OutputView.VariableView accessorNewVariable(
```

Comment:
Do we have a test for this function?

Reply:
I think most of these are just tested by baselines.